case_studies/dc2_cataloging/run_great_lake/configs/full_train_config_great_lake_exp_06-16-1.yaml

```diff
@@ -0,0 +1,165 @@
+        include_original: false
+        include_background: false
+        concat_psf_params: false
+        num_psf_params: 4  # for SDSS, 4 for DC2
+        log_transform_stdevs: []
+        use_clahe: false
+        clahe_min_stdev: 200
+        asinh_params:
+            scale: 1000
+            thresholds: [-0.3856, -0.1059, -0.0336,  0.0073,  0.0569,  0.1658,  0.6423]
+    matcher:
+        _target_: bliss.encoder.metrics.CatalogMatcher
+        dist_slack: 1.0
+        mag_slack: null
+        mag_band: 2  # SDSS r-band
+    metrics:
+        _target_: torchmetrics.MetricCollection
+        _convert_: "partial"
+        compute_groups: false
+        metrics:
+          detection_performance:
+            _target_: bliss.encoder.metrics.DetectionPerformance
+            mag_bin_cutoffs: [200, 400, 600, 800, 1000]
+            bin_unit_is_flux: true
+          detection_performance_star:
+            _target_: bliss.encoder.metrics.DetectionPerformance
+            mag_bin_cutoffs: [200, 400, 600, 800, 1000]
+            bin_unit_is_flux: true
+            filter_list:
+              - _target_: bliss.encoder.metrics.SourceTypeFilter
+                filter_type: "star"
+          detection_performance_galaxy:
+            _target_: bliss.encoder.metrics.DetectionPerformance
+            mag_bin_cutoffs: [200, 400, 600, 800, 1000]
+            bin_unit_is_flux: true
+            filter_list:
+              - _target_: bliss.encoder.metrics.SourceTypeFilter
+                filter_type: "galaxy"
+          source_type_accuracy:
+            _target_: bliss.encoder.metrics.SourceTypeAccuracy
+            flux_bin_cutoffs: [200, 400, 600, 800, 1000]
+          source_type_accuracy_star:
+            _target_: bliss.encoder.metrics.SourceTypeAccuracy
+            flux_bin_cutoffs: [200, 400, 600, 800, 1000]
+            filter_list:
+              - _target_: bliss.encoder.metrics.SourceTypeFilter
+                filter_type: "star"
+          source_type_accuracy_galaxy:
+            _target_: bliss.encoder.metrics.SourceTypeAccuracy
+            flux_bin_cutoffs: [200, 400, 600, 800, 1000]
+            filter_list:
+              - _target_: bliss.encoder.metrics.SourceTypeFilter
+                filter_type: "galaxy"
+          flux_error:
+            _target_: bliss.encoder.metrics.FluxError
+            survey_bands: ${encoder.survey_bands}
+          # galaxy_shape_error:
+          #   _target_: bliss.encoder.metrics.GalaxyShapeError
+    sample_image_renders:
+        _target_: torchmetrics.MetricCollection
+        metrics:
+          - _target_: bliss.encoder.sample_image_renders.PlotSampleImages
+            frequency: 1
+            restrict_batch: 0
+            tiles_to_crop: 1
+            tile_slen: ${simulator.prior.tile_slen}
+
+    do_data_augmentation: true
+    compile_model: false
+    double_detect: false
+    use_checkerboard: true
+
+surveys:
+   dc2:
+      split_results_dir: ${paths.dc2}/dc2_split_results
```

Comment:
Better to treat the survey directories (e.g., `${paths.dc2}`) as "read only", and write output exclusively to within `${paths.output}`. When we're doing large-scale runs, we won't have write access to the survey directories.

Reply:
change it to `${paths.output}`